xray/commands/utils.go

```diff
@@ -14,6 +15,84 @@ const (
 	BypassArchiveLimitsMinXrayVersion = "3.59.0"
 )
 
+type FilterLevel int
```

Comment:
FilterLevel --> SeverityLevel
Let's also change the related functions.

Reply:
done